builtin/rebase.c

```diff
@@ -249,6 +296,7 @@ int cmd_rebase(int argc, const char **argv, const char *prefix)
 		options.state_dir = merge_dir();
 		break;
 	}
+
```

Comment:
This should go into the commit where either that `switch ()` statement was introduced, or that `if (!options.root)`, whichever came later.